pandas/core/common.py

```diff
@@ -2463,6 +2465,36 @@ def _astype_nansafe(arr, dtype, copy=True):
     return arr.view(dtype)
 
 
+def _get_next_label(label):
```

Comment:
isn't this better in ``core/interval.py``? I agree could be sort of general, but more clear there i think.